### PR TITLE
added buybacks

### PR DESCRIPTION
**Pull request by ugolino**
# Task:
Added buybacks to trade list

## Description

- added buyback from subgraph
 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update
- [ ] Quality of life improvement

## Checklist

- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have NATSPEC'd any new functions
- [ ] If appropriate, I have properly tested my new functionality

### CODE REVIEW

File: packages/front-end/src/components/VaultTrades.tsx [hunk 1]
```diff
@@ -85,12 +100,13 @@ export const VaultTrades = () => {
     <div className="pb-8 py-12 px-8">
 
       <div className="grid grid-cols-12 text-left text-lg pb-4 px-2">
```
Comment (devey) [inline packages/front-end/src/components/VaultTrades.tsx:102]: Look good. One thing is that all of these columns are making the ui crowded, and also leading to text wrapping.
Can work on this at some point.

Comment (devey) [reply packages/front-end/src/components/VaultTrades.tsx:102]: It would also be interesting to have a filter here by type of action :)

**Review by devey — APPROVED** [review]
lgtm

File: packages/front-end/src/components/VaultTrades.tsx (hunk 1 above)
Comment (ugolino) [reply packages/front-end/src/components/VaultTrades.tsx:102]: ok. will merge now and we can clean it and have filter later

**Pull request merged**